test/integration-tests/CheckOwnership.test.ts

```diff
@@ -87,18 +87,13 @@ describe('Check Ownership', () => {
       const { commands, inputs } = planner
 
       const ownerBefore = await cryptoCovens.ownerOf(params.offer[0].identifierOrCriteria)
-      const ethBefore = await ethers.provider.getBalance(alice.address)
-      const receipt = await (
-        await router['execute(bytes,bytes[],uint256)'](commands, inputs, DEADLINE, { value })
-      ).wait()
+      await expect(
+        router['execute(bytes,bytes[],uint256)'](commands, inputs, DEADLINE, { value })
+      ).to.changeEtherBalance(alice, value.mul(-1))
```

Comment:
Wow didn't know you could do this, cool! One thing - does this check for gasSpent as well? Or does it do it already from the txn receipt

Reply:
It includes checking the gasSpent if the account you provide is the one that sent the transaction - and ignores gas otherwise